src/ui/components/Explorer/Explorer.js

```diff
@@ -31,6 +32,31 @@ import {type TimelineCredParameters} from "../../../analysis/timeline/params";
 
 import NodeRow from "./NodeRow";
 
+const styles = StyleSheet.create({
+  combobox: {margin: "0px 32px 16px"},
+  menuHeader: {fontWeight: "bold"},
+  divider: {backgroundColor: "#F20057", height: "2px"},
```

Comment:
Nit: I see you changed the divider color to gray. I rather liked the pink underline. 😊️ It'd be really cool if it changed colors when hovered over.

![image](https://user-images.githubusercontent.com/36084652/93012719-5f6a8e00-f557-11ea-878d-e43510cd3a66.png)

Reply:
I don't think I changed this... but ultimately the color scheme will be decided by Faruk and the design team.